java/src/main/java/org/whispersystems/libsignal/IdentityKeyPair.java

```diff
@@ -1,55 +0,0 @@
-/**
```

Comment:
Since IdentityKey has been replaced by ECPublicKey, we can just use ECKeyPair instead of IdentityKeyPair.